File: ruby/prob3.rb
#Work in progress`
#What is the largest prime factor of the number 600851475143 ?
#
#
def prime_factor(n)
   for i in 2..n
       if n % i == 0
           i = i / n
           puts #(i)
       end
   end
end

num = 600851475143

prime_factor(num)

File: python/prob3.py
#!/usr/bin/env python
#What is the largest prime factor of the number 600851475143?

num = 600851475143
prime = 0

def getprime(p):	#find prime number 
   for n in range(2,p):
       if p % n == 0:
          p = p / n
          print(n)

getprime(num)

# to do
# make it so that the program terminates
